samples/videoserver/pcstats.py

```diff
@@ -0,0 +1,132 @@
+##+--------------------------------------------------------------------------
+##
+## videoserver - (c) 2023 Dave Plummer.  All Rights Reserved.
+##
+## File:        pcstats.py - Pushes text about CPU use, etc, to LED matrix
+##
+## Description:
+##
+##    Creates a bitmap of text that shows CPU usage, memory usage, etc
+##    compresses it, and sends it over WiFi to NightDriverStrip
+##
+## History:     Oct-21-2023     davepl      Created
+##
+##---------------------------------------------------------------------------
+
+# Dependencies
+
+from PIL import Image, ImageDraw, ImageFont     # python3 -m pip install pillow
+import sys
+import socket
+import time
+import struct
+import zlib
+import datetime
```

Comment:
I think we can drop the import of sys and struct here, as they don't seem to be used.